### PR TITLE
Feature/reblog screen

### PR DESCRIPTION
**Pull request by mistikk**

### CODE REVIEW

File: src/components/postCard/view/postCardView.js [hunk 1]
```diff
@@ -69,6 +69,14 @@ class PostCardView extends Component {
     handleOnVotersPress(content.active_votes);
   };
 
+  _handleOnReblogsPress = () => {
+    const { handleOnReblogsPress, content } = this.props;
+
+    if (content.reblogs.length > 0) {
+      handleOnReblogsPress(content.reblogs);
```
Comment (ue) [inline src/components/postCard/view/postCardView.js]: use get please

File: src/components/postView/container/postDisplayContainer.js [hunk 2]
```diff
@@ -51,6 +51,20 @@ class PostDisplayContainer extends Component {
...
+  _handleOnReblogsPress = reblogs => {
+    const { navigation, post } = this.props;
+
+    if (reblogs.length > 0) {
+      navigation.navigate({
+        routeName: ROUTES.SCREENS.REBLOGS,
+        params: {
+          reblogs,
+        },
+        key: post.permlink + Math.random(),
```
Comment (ue) [inline src/components/postView/container/postDisplayContainer.js]: random is not sugested way

Comment (ue) [reply src/components/postView/container/postDisplayContainer.js]: you can set as post.permlink + reblogs count
 also use get

File: src/screens/voters/screen/votersScreen.js [hunk 3]
```diff
@@ -9,90 +9,46 @@ import { BasicHeader } from '../../../components/basicHeader';
...
+            dropdownIconName="arrow-drop-down"
+            options={filterOptions.map(item =>
+              intl.formatMessage({
+                id: `voters_dropdown.${item}`,
+              }),
+            )}
+            defaultText={intl.formatMessage({ id: `voters_dropdown.${filterOptions[0]}` })}
+            onDropdownSelect={handleOnVotersDropdownSelect}
+          />
+          <VotersDisplay key={Math.random()} votes={filterResult || data} />
```
Comment (ue) [inline src/screens/voters/screen/votersScreen.js]: please remove math thing use correct way

File: src/utils/postParser.js [hunk 4]
```diff
@@ -67,8 +67,8 @@ export const parsePost = async (post, currentUserName, isPromoted) => {
     });
   }
 
-  const postReblogs = await getPostReblogs(post);
-  post.reblogCount = postReblogs.length;
+  post.reblogs = await getPostReblogs(post);
+  post.reblogCount = post.reblogs.length;
```
Comment (ue) [inline src/utils/postParser.js]: use get

**Comment by mistikk** [comment]
@ue updated

**Pull request merged**